gridcomps/History3G/HistoryGridComp.F90

```diff
@@ -53,13 +58,35 @@ subroutine setServices(gridcomp, rc)
          collection_name = ESMF_HConfigAsString(iter, _RC)
          child_hconfig = make_child_hconfig(hconfig, collection_name, _RC)
          child_name = make_child_name(collection_name, _RC)
-         call MAPL_GridCompAddChild(gridcomp, child_name, user_setservices(collection_setServices), child_hconfig, _RC)
-         !call ESMF_HConfigDestroy(child_hconfig, _RC)
+
+         call get_child_timestep(child_hconfig, timeStep, _RC)
+         child_spec = ChildSpec(user_setservices(collection_setServices), hconfig=child_hconfig, timeStep=timeStep, refTime=refTime)
```

Comment:
@tclune, `refTime` appears to be uninitialized.

Reply:
Yes - by design.    We don't have a test that covers that case yet.    The variable is unallocated for now, which is equivalent to "use default from parent" which is 99% of what we want anyway.  So no harm.